src/Landing/redux/reducers/LatestArticles.jsx

```diff
@@ -0,0 +1,20 @@
+import { FETCH_LATEST_POSTS } from '../types';
+
+const initialState = {
+  items: [],
+  item: {},
+};
+
+const LatestArticleReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_LATEST_POSTS:
+      return {
+        ...state,
+        items: action.payload,
+      };
+    default:
+      return state;
+  }
+};
+
+export default LatestArticleReducer;
```

Comment:
This file could use a better name like latestArticlesReducer.js

Reply:
Done renaming appropriately